crowdin.yml

```diff
@@ -0,0 +1,8 @@
+project_id_env: CROWDIN_PROJECT_ID
+api_token_env: CROWDIN_PERSONAL_TOKEN
```

Comment:
👏🏼

Reply:
should this need to be added in `.env`?

Reply:
I looked into it a bit, and as far as I can tell, crowdin.yml is not used in any scenario when .env is in effect.
crowdin.yml will be used when:
1. used by github workflows (crowdin-download.yml and crowdin-upload.yml), which provide CROWDIN_* constants from github secrets.
2. running "yarn sync", etc. from command line. In that case, it seems like `.env` is not in effect anyway, so the developer still needs to define environment variables in their own shell.

So i think that it is not needed in .env.

Reply:
Gotcha 👍🏼 can we add in the docs about it 🙏🏼